error running example

Hi,

I'm trying to run the example and getting the following error after pulling the repo and doing : 'npm install'

npm ERR! peerinvalid The package loopback-datasource-juggler does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer loopback@2.0.0-beta3 wants loopback-datasource-juggler@2.0.0-beta1
npm ERR! peerinvalid Peer loopback-connector-mongodb@1.2.6 wants loopback-datasource-juggler@1.x.x

npm ERR! System Darwin 13.2.0
npm ERR! command "node" "/usr/local/bin/npm" "install"
npm ERR! cwd /Users/xxxx/loopback-example-full-stack
npm ERR! node -v v0.10.28
npm ERR! npm -v 1.4.9
npm ERR! code EPEERINVALID
npm ERR! 

What do I need to do to get the full stack example running? Any help much appreciated. Thanks.

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -33,7 +33,9 @@ gulp.task('run', function(cb) {
     })
     .on('change', function() {
       // TODO(ritch) only build the package that changed
-      findAndBuild('*');
+      findAndBuild('*', function(err) {
+        cb(err);
+      });
     })
     .on('restart', function () {
       console.log('restarted!');

--- a/html5/configure.js
+++ b/html5/configure.js
@@ -7,6 +7,8 @@ var browserify = require('browserify');
 var sh = require('shelljs');
 var BootConfigLoader = require('loopback-boot').ConfigLoader;
 
+var buildDir = path.resolve(__dirname, 'build');
+
 exports.global = function(env, global) {
   // routes
   global.routes = {
@@ -44,7 +46,7 @@ exports.global = function(env, global) {
   if(!isDev(env)) global.bundle += '.min';
   global.bundle += '.js';
   global.html5Views = path.join(__dirname, 'views');
-  global.html5Bundle = path.join(__dirname, 'build', global.bundle);
+  global.html5Bundle = path.join(buildDir, global.bundle);
   global.bundleURL = '/' + global.bundle;
 }
 
@@ -64,6 +66,14 @@ exports.local = function configure(env, global, local) {
 
 exports.build = function(env, global, local, cb) {
   async.waterfall([
+    function createBuildDir(next) {
+      fs.exists(buildDir, function(yes) {
+        if (yes)
+          next();
+        else
+          fs.mkdir(buildDir, next);
+      });
+    },
     function(next) {
       async.parallel([
         function(next) {

